examples/async/async.js

```diff
@@ -80,6 +83,12 @@ search.addWidget(
       item: hitTemplate
     },
     transformData: function(hit) {
+      var result = search.helper.lastResults;
+      var offset = result.hitsPerPage * result.page;
+
+      hit.queryID = result.queryID;
+      hit.hitPosition = offset + hit.__hitIndex + 1;
```

Comment:
these two should probably have two underscores so that they don't collapse with user's attributes